api/docs/docs/data.md

```diff
@@ -0,0 +1,210 @@
+ 3. An outlier detection filter is applied to the timeseries, removing any data points that
+    seem improbable given recent case numbers.  Many times this is due to a backfill of
+    previously unreported cases.
+
+**Where to access**  
+* CSV column names: ``actuals.newCases``
+* JSON file fields: ``actuals.newCases``, ``actualsTimeseries.*.newCases``
+
+### Case Density
+
+  The number of cases per 100k population calculated using a 7-day rolling average.
+
+**Where to access**  
+* CSV column names: ``metrics.caseDensity``
+* JSON file fields: ``metrics.caseDensity``, ``metricsTimeseries.*.caseDensity``
+
+### Infection Rate
+
+  R_t, or the estimated number of infections arising from a typical case.
+
+**Where to access**  
+* CSV column names: ``metrics.infectionRate``
+* JSON file fields: ``metrics.infectionRate``, ``metricsTimeseries.*.infectionRate``
+
+### Infection Rate Ci90
+
+  90th percentile confidence interval upper endpoint of the infection rate.
+
+**Where to access**  
+* CSV column names: ``metrics.infectionRateCI90``
+* JSON file fields: ``metrics.infectionRateCI90``, ``metricsTimeseries.*.infectionRateCI90``
+
+
+
+## Tests
+
+### Positive Tests
+
+  Cumulative positive test results to date
+
+**Where to access**  
+* CSV column names: ``actuals.positiveTests``
+* JSON file fields: ``actuals.positiveTests``, ``actualsTimeseries.*.positiveTests``
+
+### Negative Tests
+
+  Cumulative negative test results to date
+
+**Where to access**  
+* CSV column names: ``actuals.negativeTests``
+* JSON file fields: ``actuals.negativeTests``, ``actualsTimeseries.*.negativeTests``
+
+### Test Positivity Ratio
+
+  Ratio of people who test positive calculated using a 7-day rolling average.
+
+**Where to access**  
+* CSV column names: ``metrics.testPositivityRatio``
+* JSON file fields: ``metrics.testPositivityRatio``, ``metricsTimeseries.*.testPositivityRatio``
+
+### Test Positivity Ratio Details
+
+  
+
+**Where to access**  
+* CSV column names: ``metrics.testPositivityRatioDetails``
+* JSON file fields: ``metrics.testPositivityRatioDetails``, ``metricsTimeseries.*.testPositivityRatioDetails``
+
+
+
+## Hospitalizations
+
+### Icu Beds
+
+  Information about ICU bed utilization details.
+
+Fields:
+ * capacity - Current staffed ICU bed capacity.
+ * currentUsageTotal - Total number of ICU beds currently in use
+ * currentUsageCovid - Number of ICU beds currently in use by COVID patients.
+ * typicalUsageRate - Typical ICU utilization rate.
+
+**Where to access**  
+* CSV column names: ``actuals.icuBeds``
+* JSON file fields: ``actuals.icuBeds``, ``actualsTimeseries.*.icuBeds``
+
+### Hospital Beds
+
+  Information about acute bed utilization details.
+
+Fields:
+ * capacity - Current staffed acute bed capacity.
+ * currentUsageTotal - Total number of acute beds currently in use
+ * currentUsageCovid - Number of acute beds currently in use by COVID patients.
+ * typicalUsageRate - Typical acute bed utilization rate.
```

Comment:
Seeing the "fields" section here made me wonder whether we should have an "inputs" section. Like for all the topline metrics do we want to show the inputs we use for each? 
 
e.g. ICU Capacity Ratio

Inputs:

- capacity - current staffed ICU bed capacity
- currentUsagetotal - etc.